lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -686,4 +695,16 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
             this.dataRowsChanged = null;
         }
     }
+
+    datatableLayoutFix() {
+        const maxGalleryRows = 7;
```

Comment:
is this a magic number?

Reply:
A little bit, it has to be as big as the number of maximum columns that are going to be displayed. We can add a comment or think of another solution.

Reply:
this number I guess works only for a certain display size